src/components/NavBar/NavBar.tsx

```diff
@@ -8,7 +8,7 @@ import NavbarRoutes from "Props/navbar/navbar"
 const NavThemes = {
```

Comment:
I know this wasn't part of the original plan but I've been playing around and I believe might be helpful just a background safety measure. But we can add the logout button if the isAuthenticated only and have no login. This is just in case someone logs in and we need a way to log them out.